src/templates/core/OpenAPI.hbs

```diff
@@ -15,6 +66,27 @@ export type OpenAPIConfig = {
 	USERNAME?: string | Resolver<string> | undefined;
 	VERSION: string;
 	WITH_CREDENTIALS: boolean;
+	interceptors: {
```

Comment:
should this be INTERCEPTORS?

Reply:
Uppercase is a terrible interface for users, all caps should be reserved only for constants